openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -11,10 +17,63 @@
     deserialize_bytes_to_event_data,
     serialize_event_data_to_bytes,
 )
+from openedx_events.learning import signals  # pylint: disable=reimported
+from openedx_events.tests.utils import FreezeSignalCacheMixin
+from openedx_events.tooling import OpenEdxPublicSignal
+
+# If a signal is explicitly not for use with the event bus, add it to this list
+KNOWN_UNSERIALIZABLE_SIGNALS = ["org.openedx.learning.discussions.configuration.changed.v1"]
+
+# TODO (EventBus): figure out how to automatically pull new signals in
+
+
+def generate_test_event_data_for_data_type(data_type):
+    """
+    Generates test data automatically by filling in dummy data for basic data types (int/bool/str)
+    Recursively breaks down the classes for nested classes
+
+    Arguments:
+        data_type: The type of the data which we are generating data for
+
+    Returns:
+        dtdict: A data dictionary containing dummy data for all attributes of the class
```

Comment:
Just provide the type here. "dtdict" is not really meaningful externally.
```suggestion
        (dict): A data dictionary containing dummy data for all attributes of the class
```